code/upstream-integration-tests/src/androidTest/java/com/adobe/marketing/mobile/integration/util/TestSetupHelper.kt

```diff
@@ -0,0 +1,101 @@
+package com.adobe.marketing.mobile.integration.util
+
+import com.adobe.marketing.mobile.Event
+import com.adobe.marketing.mobile.util.TestConstants
+import com.adobe.marketing.mobile.util.TestHelper
+
+object TestSetupHelper {
+    /**
+     * Returns the environment file ID for the provided edgeEnvironment value.
+     *
+     * @param edgeEnvironment The edgeEnvironment value to use.
+     * @return The environment file ID for the provided edgeEnvironment value.
+     */
+    fun getEnvironmentFileID(edgeEnvironment: String): String {
+        when (edgeEnvironment) {
+            "prod" -> {
+                return "94f571f308d5/6b1be84da76a/launch-023a1b64f561-development"
+            }
+
+            "preProd" -> {
```

Comment:
looks like this string is different than iOS, the actual raw string should be "pre-prod"